### PR TITLE
adding support for linux/ppc64le arch

### LINKED ISSUE #12350: add ppc64le linux/ppc64le arch in CI
Hi there,
I've successfully built and tested the collector-contrib on ppc64le VM.

- rasing PR for the same.
[PR link](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/12166)

### PR DESCRIPTION
**Pull request by adilhusain-s**
Hi there, 
I've successfully built and tested the opentelemetry-collector-contrib on power VM. 
* adding code changes to generate binaries for ppc64le arch in github actions.

please review and merge.

Resolves #12350

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 # v0.55.0
 
 ## 🛑 Breaking changes 🛑
+- Add `linux-ppc64le` architecture to cross build tests in CI
```
Comment (jpkrohling) [inline CHANGELOG.md:10]: The changelog in this repo works differently, check the contribution guidelines. In short, add a file to ./unreleased. That said, I don't think we should have this listed in the changelog.

**Comment by adilhusain-s** [comment]
@jpkrohling
thanks, please review.

**Review by jpkrohling — APPROVED** [review]

**Comment by adilhusain-s** [comment]
please run the workflow. excluding the ppc64le arch for darwin and windows.

**Comment by jpkrohling** [comment]
I assume the problem is then that a specific component can't be compiled on those platforms. Would you please open a ticket to address that? We should record this somewhere, as it would certainly leave us wondering in the future why core has the whole set, and contrib has some exclusions.

**Comment by adilhusain-s** [comment]
the collector-core uses goreleaser to build the binaries and goreleaser is smart enough to understand that ppc64le arch isn't supported on windows and darwin. on the other hand, for collector-contrib we're cross compiling the binaries using the golang compiler itself. 
no, the collector-core doesn't have binaries for darwin/ppc64le or windows/ppc64le. 
you may check the latest run of the cross build  workflow.
[workflow link](https://github.com/open-telemetry/opentelemetry-collector/runs/7252770800?check_suite_focus=true)

**Comment by jpkrohling** [comment]
Got it, thanks for the clarification!

**Comment by adilhusain-s** [comment]
Please review and merge.

**Comment by jpkrohling** [comment]
@djaglowski, shouldn't this have failed due to the lack of changelog entries?

**Comment by adilhusain-s** [comment]
@dmitryax 
could you   please also review and approve this PR?
feel free to suggest any changes if required.

**Comment by djaglowski** [comment]
@jpkrohling, this appears to be due to #12194. The problem was resolved, but a rebase would be necessary to pick up the fix.

@adilhusain-s, please rebase and add a changelog entry according to `CONTRIBUTING.md`.

**Comment by adilhusain-s** [comment]
@djaglowski
thanks for  the review,

does this line look good for change log?
- Add `linux-ppc64le` architecture to cross build tests in CI
-

**Comment by jpkrohling** [comment]
Yes, I think that line is fine.

**Comment by adilhusain-s** [comment]
@jpkrohling
please run the workflows.

File: CHANGELOG.md (hunk 1 above)
Comment (djaglowski) [reply CHANGELOG.md:10]: @adilhusain-s, here are the instructions for adding a changelog entry: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#adding-a-changelog-entry

Please remove this line and add an entry according to these instructions.

**Comment by adilhusain-s** [comment]
@djaglowski 
thanks for pointing me in to right direction, 
I've created the yaml file under unrelease folder describing the change. 
please review.

**Review by djaglowski — APPROVED** [review]
Thank you @adilhusain-s

**Review by codeboten — APPROVED** [review]

**Pull request merged**